.editorconfig

```diff
@@ -0,0 +1,11 @@
+[*]
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_size = 4
+
+[*.js]
+max_line_length = 110
+
+[package*.json]
+indent_size = 2
```

Comment:
This seems to generally match the way existing files are written, but say the word if something here should be changed!

(If you aren’t familiar with `.editorconfig`, you can read more at https://editorconfig.org — it’s basically a place to set common settings in a way that’s compatible with most editors. Full disclosure: I maintain the plugin for TextMate.)